examples/for-tests/src/App.js

```diff
@@ -221,7 +221,7 @@ let recipeList = [
 ];
 
 const testContext = {
-    disableDefaultImplementation: getQueryParams("disableDefaultImplementation") === "true",
+    disableDefaultUI: getQueryParams("disableDefaultImplementation") === "true",
```

Comment:
Consider changing the queryParams key as well

Reply:
Doing that in this PR: https://github.com/supertokens/supertokens-auth-react/pull/426 to keep changes separated